File: plugin/src/client/sc/plugin2019/PlayerType.java
package sc.plugin2019;

/** Die Rolle eines Clients */
public enum PlayerType {
  // NOTE: Order is important, see {@link GamePreparation#nextHumanPlayerId()}.
  // Player one needs to be declared before player two.
  NONE, OBSERVER, PLAYER_ONE, PLAYER_TWO
}

File: plugin/src/client/sc/plugin2021/PlayerType.kt
package sc.plugin2021

import com.thoughtworks.xstream.annotations.XStreamAlias

/** Beschreibt den Typ eines Clients. */
@XStreamAlias(value = "playertype")
enum class PlayerType {
    NONE,
    OBSERVER,
    PLAYER_ONE,
    PLAYER_TWO,
}